code/modules/mob/mob_helpers.dm

```diff
@@ -30,6 +30,44 @@
 /mob/living/carbon/human/isMonkey()
 	return istype(species, /datum/species/monkey)
 
+
+/proc/ishuman_species(A)
+	if(istype(A, /mob/living/carbon/human) && (A:get_species() == "Human"))

```

Comment:
Replace the istype() here with ishuman(A). Although it would basically amount to the same thing. Eeeeeeeeeeeeeeeeeeh. Eeeeh. eh. Eh. eh. EHH. eh.

/ASCIIshrug.